src/signal_scope/plotwidget.cpp

```diff
@@ -289,6 +321,17 @@ void PlotWidget::onResetYAxisScale()
     if (this->signalIsVisible(signalHandler))
     {
       QRectF signalBounds = signalHandler->signalData()->computeBounds();
+      if (signalBounds.top() == signalBounds.bottom()){  // in case of a constant signal
+        if (signalBounds.bottom() >= 0.0)
+          signalBounds.setBottom(signalBounds.bottom()*1.1);
+        else
+          signalBounds.setBottom(signalBounds.bottom()*0.9);
+
+        if (signalBounds.top() >= 0.0)
+          signalBounds.setTop(signalBounds.top()*0.9);
+        else
+          signalBounds.setTop(signalBounds.top()*1.1);
```

Comment:
possibly a simpler way to express this-

double margin = std::abs(signalBounds.top()*0.1)
signalBounds.setTop(signalBounds.top() + margin);
signalBounds.setBottom(signalBounds.bottom() - margin);